<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>698</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>464</width>
    <height>698</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>464</width>
    <height>698</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#LoginBtn {
background-color: transparent;
border-image: url(:Autentificare_off.png);
background: none;
border: none;
background-repeat: none;
}
#LoginBtn:hover
{
border-image: url(:Autentificare_on.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>570</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#RegisterBtn {
background-color: transparent;
border-image: url(:Inregistrare_off.png);
background: none;
border: none;
background-repeat: none;
}
#RegisterBtn:hover
{
border-image: url(:Inregistrare_on.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logo {
background-color: transparent;
border-image: url(:Logo.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#UsernameLabel
{
color:#fff;
background-color: transparent;
background:none;
}</string>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#PasswordLabel
{
color:#fff;
background-color: transparent;
background:none;
}</string>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEditPassword">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#textEditPassword
{
color:#fff;
padding-right: 40px;
border: 2px solid white;
border-radius:10px;
}

#textEditPassword:focus
{
border: 2px solid yellow;
}</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Introduceti parola</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEditUserName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#textEditUserName
{
color:#fff;
padding-right: 40px;
border: 2px solid white;
border-radius:10px;
}
#textEditUserName:focus
{
border: 2px solid yellow;
}</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="placeholderText">
     <string>Introduceti mail-ul</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>400</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#checkBox
{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
}

QCheckBox::indicator:unchecked {
    image: url(:showWhite.png);
}

QCheckBox::indicator:checked {
    image: url(:hideWhite.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mail_icon">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>275</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mail_icon
{
background-color: transparent;
border-image: url(:mail_icon.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
